mobile/app.json

```diff
@@ -39,10 +39,18 @@
         "googleMaps": {
           "apiKey": "AIzaSyDtbPnRIXbOOpsvEQ-a6nrLnZSjofObpXc"
         }
-      }
+      },
+      "permissions": [
+        "android.permission.RECORD_AUDIO"
```

Comment:
New feature? 👀

Reply:
wops, I don't know why this permission was added :eyes: